### ISSUE TITLE
Error when running make build latest commits

### ISSUE THREAD

**Issue by PierrunoYT** [OpenDevin/OpenDevin#757]
![grafik](https://github.com/OpenDevin/OpenDevin/assets/95778421/be349c69-484c-486f-b359-3b71c63e6f35)

**Comment by nullnuller** [comment]
I am getting error as well with corepack in the latest pull. What's wrong?

```Current Node.js version is 21.7.2, corepack is supported.
Internal Error: EACCES: permission denied, symlink '../lib/node_modules/corepack/dist/pnpm.js' -> '/usr/bin/pnpm'
    at async Object.symlink (node:internal/fs/promises:997:10)
    at async EnableCommand.generatePosixLink (/usr/lib/node_modules/corepack/dist/lib/corepack.cjs:23591:5)
    at async EnableCommand.execute (/usr/lib/node_modules/corepack/dist/lib/corepack.cjs:23575:11)
    at async EnableCommand.validateAndExecute (/usr/lib/node_modules/corepack/dist/lib/corepack.cjs:21173:22)
    at async _Cli.run (/usr/lib/node_modules/corepack/dist/lib/corepack.cjs:22148:18)
    at async Object.runMain (/usr/lib/node_modules/corepack/dist/lib/corepack.cjs:24266:12)
make: *** [Makefile:29: build] Error 1
```

**Comment by dorbanianas** [comment]
- This is considered solved using this PR #796 👌

Enjoy 😁!
